### PR TITLE
add --version option

### PR DESCRIPTION
**Pull request by rcerven**
osbs issue #485 

adds version option for osbs client

### CODE REVIEW

File: osbs/cli/main.py [hunk 1]
```diff
@@ -457,6 +458,11 @@ def str_on_2_unicode_on_3(s):
 
 
 def cli():
+    try:
+        version = pkg_resources.get_distribution("osbs-client").version
+    except pkg_resources.DistributionNotFound:
```
Comment (lcarva) [inline osbs/cli/main.py]: In what scenario would this exception be triggered? When I have it installed from git, `get_distribution` is still able to find it properly.

Comment (rcerven) [inline osbs/cli/main.py]: it is when it is not installed

**Comment by twaugh** [comment]
Does it make sense to test it?

**Comment by rcerven** [comment]
well all you have to test is osbs --version :)

**Comment by twaugh** [comment]
I was thinking about unit testing really...

**Comment by rcerven** [comment]
in that case it doesn't make sense to test

**Pull request merged**